js/indicators/type_of_categories.js

```diff
@@ -0,0 +1,65 @@
+function display_type_of_categories(lines, elementId) {
+    let categories_h = {}
```

Comment:
proposal to rename variable to `male`/`female` (makes it easier to read imo) and add a third one for those who identify as neither